.azure/OneBranch.Publish.yml

```diff
@@ -78,17 +89,25 @@ stages:
           runId: $(resources.pipeline.onebranch.runID)
           artifact: drop_package_linux_distribution_openssl3
           path: $(Build.SourcesDirectory)/artifacts/signed/openssl3
+      - task: DownloadSecureFile@1
+        name:  pmcv4cert
+        displayName: 'Download cert for PMC v4'
+        inputs:
+          secureFile: 'auth.pem'
       - task: Docker@2
         displayName: Login to ACR
         inputs:
           command: login
           containerRegistry: msquicdockerregistry
       - ${{ each repo in parameters.opensslrpmrepos }}:
```

Comment:
Instead of a for each, can we do a matrix strategy and parallelize?

Reply:
In the internal repo, I am using 4 jobs to make them run in parallel.